base/monitoring/jaeger/jaeger.Deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     spec:
       containers:
         - name: jaeger
-          image: index.docker.io/sourcegraph/jaeger-all-in-one:5.1.2@sha256:7189ff3a1ecdcb2e0f4f8208078bf67f3a04a460469b30cc09ea537e94764098
+          image: index.docker.io/sourcegraph/jaeger-all-in-one:5.1.3@sha256:7189ff3a1ecdcb2e0f4f8208078bf67f3a04a460469b30cc09ea537e94764098
```

Comment:
the sha didn't change? 🤔

Reply:
oh, apparently that's for multiple images, maybe a cool side-effect of bazel?

Reply:
I believe so. 

I thought it suspect also, but apparently, it's not a new occurrence. Check out [5.1.2](https://github.com/sourcegraph/deploy-sourcegraph-k8s/pull/38/files)